src/pseudo/load_psp.jl

```diff
@@ -28,8 +28,10 @@ function load_psp(key::AbstractString)
         if startswith(identifier, datadir_psp())
             identifier = identifier[length(datadir_psp())+1:end]
         end
-    else
-        # Not a file: Treat as identifier, add extension if needed
+    elseif startswith(lowercase(key), r"http://|https://")  # Key is a URL .. download it
```

Comment:
Is there so standard library function to check for an URI or URL?

Reply:
Unfortunately not, we could add `URIs.jl` as a dependency for this.

Reply:
Nah that's a bit over the top, ok let's leave as is. But make a comment in the docstring that this will be dropped when we move to the artefacts system, so we make no guarantees towards the future.